### PR TITLE
Addresses #2957 revision of distal pulmonary hierarchy

### LINKED ISSUE #2957: Edit pulmonary hierarchy
The distal pulmonary hierarchy needs the following edits: 
- Remove `bronchiole SubclassOf 'part of' some 'pulmonary lobule'`. Only distal bronchioles or respiratory bronchioles are part of a (secondary) pulmonary lobule
- Remove `'pulmonary acinus' SubclassOf 'pulmonary lobule'`. This is based on the premise that 'pulmonary acinus' is an exact synonym of 'primary pulmonary acinus'. However, according to this [reference](https://doi.org/10.53347/rID-8763), "A primary pulmonary lobule is defined as the lung unit distal to the respiratory bronchioles. It is significantly smaller than an [acinus](https://radiopaedia.org/articles/pulmonary-acinus?lang=us), and is composed of alveolar ducts, alveolar sacs and alveoli."
- Add `'pulmonary acinus' SubclassOf 'part of' some 'secondary pulmonary lobule'`. According to this [reference](https://doi.org/10.53347/rID-8761), "Secondary pulmonary lobules represent a cluster of up to 30 [acini](https://radiopaedia.org/articles/pulmonary-acinus?lang=us)"
- Remove `'pulmonary lobule' SubclassOf 'has part' some 'terminal bronchiole'`. In the ticket #2952 'terminal bronchiole' will be SubclassOf 'part of' some 'secondary bronchiole' and in the ticket #2956 'pulmonary lobule' will be merged with 'secondary pulmonary bronchiole'.
- - Add `'terminal bronchiole' SubclassOf ' distally connected to' some 'respiratory bronchiole'`. See diagram below.
- Add `'terminal bronchiole' SubclassOf ' distally connected to' some 'pulmonary acinus'`. 'terminal bronchiole' is 'distally connected to' some 'respiratory bronchiole', and respiratory bronchiole are 'part of' some 'pulmonary acinus'. See diagram
image

- Add `'pulmonary acinus' SubclassOf 'connected to' some 'terminal bronchiole'`. See previous point.

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #2957 revision of distal pulmonary hierarchy
Only hierarchy is revised, no text definitions or dbxrefs. These will be addressed in a separate ticket.

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -43863,6 +43862,8 @@ xref: SCTID:278981008
 xref: UMLS:C0225666 {source="ncithesaurus:Terminal_Bronchiole"}
 xref: Wikipedia:Terminal_bronchiole
 is_a: UBERON:0002186 ! bronchiole
+relationship: distally_connected_to UBERON:0002188 ! respiratory bronchiole
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:43865]: Consider if 'continuous with' is a more accurate object property.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -43863,6 +43862,8 @@ xref: SCTID:278981008
...
+relationship: distally_connected_to UBERON:0008874 ! pulmonary acinus
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:43866]: The text def of connected to:
"a is connected to b if and only if a and b are discrete structure, and there exists some connecting structure c, such that c connects a and b"

In this relationship, what is 'c'? Can it be 'respiratory bronchiole' if 'respiratory bronchiole' is already asserted to be part of the 'pulmonary acinus'?

**Review by bvarner-ebi — COMMENTED** [review]
Kindly see inline comments.
Will approve if there are no concerns expressed from assignee's side.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:43865]: Thank you for this comment, Bradley. There is this related ticket in Uberon #2926 and it was treated in the last Uberon call (19/06/23) about the usage of connected to and how it is not aligned with the definitions. However, it was agreed to use this object property for the pulmonary system.

**Review by bvarner-ebi — APPROVED** [review]

File: src/ontology/uberon-edit.obo (hunk 2 above)
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:43866]: yes, in this case 'respiratory bronchiole' would be c and the usage of connected_to would be more aligned with the definition.

**Pull request merged**